src/ert/gui/tools/event_viewer/panel.py

```diff
@@ -3,10 +3,11 @@
 from typing import Generator
 
 from qtpy import QtCore
+from qtpy.QtCore import QObject
```

Comment:
I changed it the other way around, only doing `from qtpy import QtCore` and deleted the import below. I don't think we waste much resources by importing QtCore, the whole package must be parsed anyways, and it probably only takes up space in some symbol table. Writing `QtCore.Slot` instead of just `Slot` down in the source code makes it easyer to see that this object comes from Qt.

Reply:
Nope, I have to revert this, as it trips in other stuff:
`src/ert/gui/tools/event_viewer/panel.py:22:8: E1101: Module 'qtpy.QtCore' has no 'QObject' member (no-member)`